app/components/newsPage/threads/actions.ts

```diff
@@ -15,13 +19,23 @@ interface AddUserComment {
 }
 
 export const addUserComment = withAuth(async (user: UserSession, body: AddUserComment) => {
-  const { comment, commentType, objectId, parentCommentId } = body;
-  const author = user;
+  try {
+    const { comment, commentType, objectId, parentCommentId } = body;
+    const author = user;
 
-  const createdComment = await addComment({ author, comment, commentType, objectId, parentCommentId });
+    const createdComment = await addComment({ author, comment, commentType, objectId, parentCommentId });
 
-  return {
-    status: StatusCodes.OK,
-    data: createdComment,
-  };
+    return {
+      status: StatusCodes.OK,
+      data: createdComment,
+    };
+  } catch (err) {
+    const error: InnoPlatformError = dbError(
+      `Adding a ${CommentType} by user ${user.providerId}`,
+      err as Error,
+      user.providerId,
+    );
+    logger.error(error);
+    throw err;
```

Comment:
Since the `addComment` method already catches the error (has try, catch block), we can remove this block because it will be caught and logged within the method.
Same applies for all the methods in which the db service is called where the error is already logged.